test/trans.render.spec.js

```diff
@@ -408,3 +408,18 @@ describe('trans should not break on invalid node from translations - part2', ()
     expect(wrapper.contains(<div>&lt;hello&gt;</div>)).toBe(true);
   });
 });
+
+describe('trans getting passed tOptions.interpolation should override tOptions.interpolation', () => {
+  const stringWithHTMLTags = '<no>Hello';
+  const TestElement = () => (
+    <Trans tOptions={{ interpolation: { escapeValue: true } }}>
+      <div> {stringWithHTMLTags} </div>
```

Comment:
```suggestion
      <div>{stringWithHTMLTags}</div>
```

Maybe the extra spaces are the problem?

Reply:
Oh, I didn't even notice the spaces! 😅

Reply:
I would try this first, thanks both of you :two_hearts: